package.json

```diff
@@ -27,9 +27,9 @@
     "lint:js:fix": "eslint src --ext .js,.jsx,.ts,.tsx --fix",
     "tsc:compile": "tsc --noEmit",
     "format": "prettier --loglevel silent --write 'src/**/*.(js|ts|jsx|tsx)'",
-    "test-ext": "jest ./src/extension",
-    "test-app": "jest ./src/app",
-    "test": "jest",
+    "test:unit": "jest",
+    "test:e2e": "npx playwright test",
+    "test": "test:unit && test:e2e",
```

Comment:
```suggestion
    "test": "yarn run test:unit && yarn run test:e2e",
```

Reply:
It's actually not needed when using `yarn`. From my experience omitting this is the default when using `yarn`.

Reply:
Ah, sorry..... correcting!